config/database.yml

```diff
@@ -1,12 +1,13 @@
 default: &default
   adapter:  "mysql2"
 
-  host:     <%= ENV['VELUM_DB_HOST']     || "localhost" %>
-  port:     <%= ENV['VELUM_DB_PORT']     || 0           %>
-  username: <%= ENV['VELUM_DB_USERNAME'] || "root"      %>
-  password: <%= ENV['VELUM_DB_PASSWORD'] || ""          %>
```

Comment:
I'm not sure why we introduced these extra spaces in the first place but full free to remove them. I don't remember any style rule for this one and they look weird.